.github/workflows/run-integration-tests.yml

```diff
@@ -39,25 +39,26 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  composer create-project --prefer-dist laravel/lumen:${{ matrix.lumen }} --stability=dev --no-progress sample
+                  composer create-project --prefer-dist laravel/lumen:${{ matrix.lumen }} --stability=dev --no-progress --ignore-platform-req=php+ sample
                   cd sample
                   composer config minimum-stability dev
-                  composer update  --prefer-stable --prefer-dist --no-progress
+                  composer update  --prefer-stable --prefer-dist --no-progress --ignore-platform-req=php+
             - name: Add package from source
               run: |
                   cd sample
                   sed -e 's|"type": "project",|&\n"repositories": [ { "type": "path", "url": "../src" } ],|' -i composer.json
-                  composer require --dev "barryvdh/laravel-debugbar:*"
+                  composer require --dev --ignore-platform-req=php+ "barryvdh/laravel-debugbar:*"
             - name: Insert service provider
               run: sed -e 's|// \$app->register(App\\\Providers\\\EventServiceProvider::class);|&\n$app->register(\\Barryvdh\\Debugbar\\LumenServiceProvider::class);|' -i sample/bootstrap/app.php
 
             - name: Execute clear run
               run: |
                   cd sample
+                  mkdir -p "storage/debugbar/" && touch "storage/debugbar/foo.json"
                   php artisan debugbar:clear
             - name: Check file count in logs
               run: |
-                if [ `ls -1q "sample/storage/logs/" | wc -l` -gt 0 ];then exit 1;fi
```

Comment:
What is this change exactly?

Reply:
I'm not exactly sure on what the intended check was doing. This check was introduced with #1118 and seems to aim to check for 'succesful completion' of the `php artisan debugbar:clear` command on a fresh Laravel installation. Here 'succesful' seems to be measured by "not logging any messages".

However, this metric seems slightly off to me. In this case it was erring, because (due to https://github.com/maximebf/php-debugbar/pull/687) deprecations are being logged while running the command. There are ways to work around this, e.g. suppressing deprecations with some Laravel flag.

I've here opted to rather than trying to patch the old definition of 'succes', come up with a new one that actually feature-tests the actual command. Given that the purpose of the command is cleaning json files from the `storage/debugbar` directory, I've now added a check to ensure the command does just this. Thus the new definition of 'succesful' is now the combination of `php artisan debugbar:clear` exits with code 0 and does feature-wise what it's expected to do.